validation/validation-codes.md

```diff
@@ -78,6 +78,29 @@
+| E075 | 'The OCFL version declaration MUST be formatted according to the NAMASTE specification.' | https://ocfl.io/draft/spec/#E075
+| E076 | '[The OCFL version declaration] MUST be a file in the base directory of the OCFL Storage Root giving the OCFL version in the filename.' | https://ocfl.io/draft/spec/#E076
+| E077 | '[The OCFL version declaration filename] MUST conform to the pattern T=dvalue, where T must be 0, and dvalue must be ocfl_, followed by the OCFL specification version number.' | https://ocfl.io/draft/spec/#E077
+| E078 | '[The OCFL version declaration filename] must conform to the pattern T=dvalue, where T MUST be 0, and dvalue must be ocfl_, followed by the OCFL specification version number.' | https://ocfl.io/draft/spec/#E078
+| E079 | '[The OCFL version declaration filename] must conform to the pattern T=dvalue, where T must be 0, and dvalue MUST be ocfl_, followed by the OCFL specification version number.' | https://ocfl.io/draft/spec/#E079
+| E080 | 'The text contents of [the OCFL version declaration file] MUST be the same as dvalue, followed by a newline (\n).' | https://ocfl.io/draft/spec/#E080
+| E081 | 'OCFL Objects within the OCFL Storage Root also include a conformance declaration which MUST indicate OCFL Object conformance to the same or earlier version of the specification.' | https://ocfl.io/draft/spec/#E081
+| E082 | 'OCFL Object Roots MUST be stored either as the terminal resource at the end of a directory storage hierarchy or as direct children of a containing OCFL Storage Root.' | https://ocfl.io/draft/spec/#E082
+| E083 | 'There MUST be a deterministic mapping from an object identifier to a unique storage path.' | https://ocfl.io/draft/spec/#E083
+| E084 | 'Storage hierarchies MUST NOT include files within intermediate directories.' | https://ocfl.io/draft/spec/#E084
+| E085 | 'Storage hierarchies MUST be terminated by OCFL Object Roots.' | https://ocfl.io/draft/spec/#E085
+| E086 | 'The storage root extensions directory MUST conform to the same guidelines and limitations as those defined for object extensions.' | https://ocfl.io/draft/spec/#E086
+| E087 | 'An OCFL validator MUST ignore any files in the storage root it does not understand.' | https://ocfl.io/draft/spec/#E087
+| E088 | 'Additional files MUST NOT appear in other directories under the storage root.' | https://ocfl.io/draft/spec/#E088
+| E089 | 'If the preservation of non-OCFL-compliant features is required then the content MUST be wrapped in a suitable disk or filesystem image format which OCFL can treat as a regular file.' | https://ocfl.io/draft/spec/#E089
+| E090 | 'Hard and soft (symbolic) links are not portable and <span id="E090">MUST NOT</span> be used within OCFL Storage hierachies.' | https://ocfl.io/draft/spec/#E090
```

Comment:
There still appears to one `span` remaining.

Reply:
Darn, I didn't spot that :-(

Reply:
I just fixed it, reduces the pool of approvers but we should have enough folks to merge after the weekend I hope!